nsqd/http.go

```diff
@@ -226,6 +231,32 @@ func (s *httpServer) getTopicFromQuery(req *http.Request) (url.Values, *Topic, e
 	return reqParams, s.ctx.nsqd.GetTopic(topicName), nil
 }
 
+func (s *httpServer) doLookup(req *http.Request) (interface{}, error) {
```

Comment:
**something** should expose gossip status `/gossip/status` ?

I'm not sure how much of the `lookupd` endpoints you wanted to land here vs follow on PR's; Some translate easily, and some have some things i hadn't thought about (tombstoning, pre-creating in registry db, deleting, etc) for making this fully back nsqadmin.